src/components/ClaimErrorAlert/index.tsx

```diff
@@ -0,0 +1,145 @@
+import { AlertCard, Button, Images } from '@hyperplay/ui'
+import { useTranslation } from 'react-i18next'
+import { Chain, mantle } from 'viem/chains'
+import { ClaimError, NotEnoughGasError, WarningError } from '@/types/quests'
+import { errorIsSwitchChainError } from '@/helpers/claimErrors'
+
+import styles from './index.module.scss'
+
+const { AlertOctagon, WarningIcon } = Images
+
+const gasInformation: Record<number, { url: string }> = {
+  [mantle.id]: {
+    url: 'https://www.mantle.xyz/mnt'
+  }
+}
+
+type ClaimErrorAlertProps = {
+  error: Error | WarningError | ClaimError
+  networkName: string
+  currentChain: Chain | undefined
+  onOpenDiscordLink: () => void
+}
+
+export const ClaimErrorAlert = ({
+  error,
+  networkName,
+  onOpenDiscordLink,
+  currentChain
+}: ClaimErrorAlertProps) => {
+  const { t } = useTranslation()
+
+  if (error instanceof WarningError) {
+    return (
+      <AlertCard
+        icon={<WarningIcon />}
+        showClose={false}
+        title={error.title}
+        message={error.message}
+        variant="warning"
+        noBorderLeft={true}
+      />
+    )
+  }
+
+  if (error instanceof NotEnoughGasError) {
+    const currency = currentChain?.nativeCurrency.symbol ?? 'Unknown'
```

Comment:
hmmmmm I think `You'll need a bit of Unknown to claim your reward` won't read well

What if we did 

`You'll need a bit of this chain's gas token to claim your reward`

The problem with the ?? here though is that we'll get a bad translation string below with just substituting this string as well.

We really need 2 translation strings. 1 if symbol is undefined here and the 1 you have below if it is

Reply:
added